### PR TITLE
Upgrade and cleanup

### PR DESCRIPTION
**Pull request by pwalczysko**
1. A suggestion to upgrade the server to 5.6.7
2. A suggestion to add the install of the ``omero-demo-cleanup`` script from the master branch of the repo to have the script available on the demo server with the new changes

cc @sbesson @jburel

### CODE REVIEW

**Comment by pwalczysko** [comment]
Please see https://github.com/openmicroscopy/management_tools/issues/1675 - I cannot run the playbook on demo, not sure what changed since last time, when I was able to run it...

**Review by sbesson — COMMENTED** [review]
Changes looks good to me. As always, if upgrading the server across systems, the deployment needs to be scheduled esp. on high-availability systems. As mentioned inline, you could also possibly use the downtime requirement to upgrade other components

File: omero/learning.yml [hunk 1]
```diff
@@ -171,7 +171,7 @@
 
   vars:
     postgresql_version: "13"
-    omero_server_release: 5.6.6
+    omero_server_release: 5.6.7
     omero_web_release: 5.16.0
```
[the remark this answers is not in the document]
Comment (pwalczysko) [reply omero/learning.yml]: Thanks, added in https://github.com/ome/prod-playbooks/pull/368/commits/4998269cb9b11c9ba95fcd8640b7e88e7d9f34be

**Comment by pwalczysko** [comment]
implemented already via private repo. Closing.

**Pull request closed without merging**